### PR TITLE
Issue71 automate verification

### LINKED ISSUE #71: add specification for how to automate the control verification
This issue is to add a specification for how to develop a software that automates the procedure of verifying a control sequence against its CDL specification.

### PR DESCRIPTION
**Pull request by mwetter**
This closes #71

### CODE REVIEW

File: specification/source/verification.rst [hunk 1]
```diff
@@ -383,3 +387,306 @@ Despite these differences, the computed and the simulated control signals
...
+generated by a control control sequence specified in CDL against the time series
+of an implementation of a real controller.
+For both use cases, the precondition is that one control sequence,
+or several control sequences, are available in CDL.
+The output will be a report that describes whether the real implementation
+conforms to the CDL implementation within a user-specified error tolerance.
+The difference between the two uses cases is as follows:
+In scenario 1, the CDL model contains the controller and blocks that generate the control input
+time series.
+The time series from this CDL model will be used to test the real controller.
```
Comment (milicag) [inline specification/source/verification.rst:421]: Do you mean output of that CDL model vs. output of the real controller?

File: specification/source/verification.rst [hunk 2]
```diff
@@ -383,3 +387,306 @@ Despite these differences, the computed and the simulated control signals
...
+
+We address two use cases. Both uses cases verify conformance of the time series
+generated by a control control sequence specified in CDL against the time series
+of an implementation of a real controller.
+For both use cases, the precondition is that one control sequence,
+or several control sequences, are available in CDL.
+The output will be a report that describes whether the real implementation
+conforms to the CDL implementation within a user-specified error tolerance.
+The difference between the two uses cases is as follows:
+In scenario 1, the CDL model contains the controller and blocks that generate the control input
```
Comment (milicag) [inline specification/source/verification.rst]: Where do inputs to the blocks that generate the control input come from?

Comment (milicag) [reply specification/source/verification.rst]: It seems unclear what a controller is and what blocks are. Isn't a controller made out of blocks?

File: specification/source/verification.rst [hunk 3]
```diff
@@ -383,3 +387,306 @@ Despite these differences, the computed and the simulated control signals
...
+time series.
+The time series from this CDL model will be used to test the real controller.
+In scenario 2, data trended from a real controller will be used to
+verify the controller against its CDL specification.
```
Comment (milicag) [inline specification/source/verification.rst]: clarify that those are outputs of both CDL and "real" controller.

**Review by milicag — CHANGES_REQUESTED** [review]
To me it would be useful to be more precise when introducing the cases. It would also be good to summarize what the main difference between the cases is before providing scenarios, so that the scenarios really serve as illustrative examples.

File: specification/source/verification.rst (hunk 1 above)
Comment (mwetter) [reply specification/source/verification.rst:421]: Yes, see two sentences above. "Both uses cases verify conformance of the time series
generated by a control control sequence specified in CDL against the time series
of an implementation of a real controller."

File: specification/source/verification.rst (hunk 2 above)
Comment (mwetter) [reply specification/source/verification.rst]: Reformulated.

File: specification/source/verification.rst (hunk 3 above)
Comment (mwetter) [reply specification/source/verification.rst]: reformulated.

**Pull request merged**